src/js/components/products/detail.tsx

```diff
@@ -0,0 +1,124 @@
+import { Product } from '@/store/products/reducer';
+import { selectProduct, selectProductSlug } from '@/store/products/selectors';
+import routes from '@/utils/routes';
+
+type Props = {
+  product?: Product | null;
+  productSlug?: string;
+  doFetchProduct({ slug }: { slug: string }): Promise<any>;
+} & RouteComponentProps;
+
+const ProductDetail = ({ product, productSlug, doFetchProduct }: Props) => {
+  useEffect(() => {
+    if (productSlug && product === undefined) {
+      // Fetch product from API
+      doFetchProduct({ slug: productSlug });
+    }
+  }, [product, productSlug, doFetchProduct]);
+
+  if (!product) {
+    if (!productSlug || product === null) {
+      return <ProductNotFound />;
+    }
+    // Fetching product from API
+    return <Spinner />;
+  }
+
+  if (productSlug && productSlug !== product.slug) {
+    // Redirect to most recent product slug
+    return <Redirect to={routes.product_detail(product.slug)} />;
+  }
+
+  const productDescriptionHasTitle =
+    product.description &&
+    (product.description.startsWith('<h1>') ||
+      product.description.startsWith('<h2>'));
+  return (
+    <DocumentTitle title={`${product.name} | ${i18n.t('MetaShare')}`}>
+      <>
+        <PageHeader
+          className="page-header slds-p-around_x-large"
+          title={product.name}
+        />
+        <div className="slds-p-horizontal_x-large slds-p-top_x-small">
+          <BreadCrumb
+            trail={[
+              <Link to={routes.home()} key="home">
+                {i18n.t('Home')}
+              </Link>,
+              <div className="slds-p-horizontal_x-small" key={product.slug}>
+                {product.name}
+              </div>,
+            ]}
+          />
+        </div>
+        <div
+          className="slds-p-around_x-large
+            slds-grid
+            slds-gutters
+            slds-wrap"
+        >
+          <div
+            className="slds-col
+              slds-size_1-of-1
+              slds-medium-size_2-of-3
+              slds-p-bottom_x-large
+              slds-text-longform"
+          >
+            <Button
+              label={i18n.t('Create a Project')}
+              className="slds-size_full slds-p-vertical_xx-small"
+              variant="brand"
+              disabled
+            />
+          </div>
+          <div
+            className="slds-col
+              slds-size_1-of-1
+              slds-medium-size_1-of-3
+              slds-text-longform"
+          >
+            {!productDescriptionHasTitle && (
+              <h2 className="slds-text-heading_medium">{product.name}</h2>
+            )}
+            {/* This description is pre-cleaned by the API */}
+            {product.description && (
+              <p
+                className="markdown"
+                dangerouslySetInnerHTML={{ __html: product.description }}
+              />
+            )}
+            <a href={product.repo_url}>{i18n.t('Link to GitHub Repo')}</a>
```

Comment:
Do we need to say "link to"?

Reply:
ha, it is quite redundant for screen readers if so, and sighted users will see a color change/underline so I think it is safe to remove "link to"

Reply:
@stacyk @mirisuzanne Thoughts?

![Screenshot 2019-07-09 14 54 39](https://user-images.githubusercontent.com/552316/60915290-7d54a880-a259-11e9-944d-570b49852cfc.png)

Reply:
should the icon be the link color as well (and also inherit hover/focus color)?

Reply:
@stacyk Good call.

![Screenshot 2019-07-09 15 46 28](https://user-images.githubusercontent.com/552316/60918208-be03f000-a260-11e9-8a0e-947b98d7d663.png)